jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -88,6 +88,9 @@
     </div>
 
       <v-row justify="end" no-gutters>
+        <j-tooltip tooltipcontent="Convert composite subset to use only add mode to connect subregions">
+          <v-btn color="primary" text @click="simplify_subset" :disabled="!can_simplify">Simplify</v-btn>
+        </j-tooltip>
```

Comment:
I think we can just hide the button entirely when it's not available.  That will make a lot more sense for image viewers... if we're worried the user might be looking for it in other situations (only a single region, etc), then maybe we should show it but just update the tooltip to explain why its disabled?

```suggestion
        <j-tooltip v-if="can_simplify" tooltipcontent="Convert composite subset to use only add mode to connect subregions">
          <v-btn color="primary" text @click="simplify_subset">Simplify</v-btn>
        </j-tooltip>
```

Reply:
Makes sense, thanks!